src/url-loader/fs-url-loader.ts

```diff
@@ -24,9 +24,9 @@ import {UrlLoader} from './url-loader';
  * Resolves requests via the file system.
  */
 export class FSUrlLoader implements UrlLoader {
-  root: string|undefined;
+  root: string;
 
-  constructor(root?: string) {
+  constructor(root: string) {
```

Comment:
Also, we might want to pick between #L65 and #L71 for how to handle undefined roots.